### DISCUSSION TITLE
QScrollArea scroll to bottom?

### DISCUSSION THREAD

**Discussion by lukepower** [quasarframework/quasar#13746] [General - Components / Directives / etc] [answered]
Hello everyone,

I am currently developing an application where I have to show several log windows, and so far I am doing it with a QScrollArea filled with <li> elements, one for each log line. All good, but I'd like it to scroll to the bottom as soon as a new element has been added. Is there a way to do so for each log?

Thanks
Lukas

**Comment by metalsadman** [comment]
QScrollArea has methods you can use to scroll to an element, you can use a template ref or id's on your log elements then target that for the scrolling.

**Comment by pratik227** [comment, marked as answer]
Codepen - https://codepen.io/Pratik__007/pen/poamazr
